addons/web/static/src/scss/utilities_custom.scss

```diff
@@ -206,3 +206,7 @@ $utilities: map-merge(
 .smaller {
     font-size: $o-font-size-base-smaller;
 }
+
+.text-prewrap {
+    white-space: pre-wrap !important;
+}
```

Comment:
This does work but, since we upgraded to Bootstrap 5, the utilities API can be used to generate this class instead of hardcoding it.

Reply:
Done! Thank you :slightly_smiling_face: